eng/dotnet-build/build.ps1

```diff
@@ -60,6 +60,7 @@ $dotnetArguments += "$PSScriptRoot/dotnet-build.proj"
 # Then remaining arguments.
 $dotnetArguments += "/p:Configuration=$configuration"
 $dotnetArguments += "/p:DotNetBuildRepo=true"
+$dotnetArguments += "/p:DotNetBuildFromSource=true"
```

Comment:
Source build would never be used for a PowerShell script since that is for Windows. Source build isn't supported for Windows.

Reply:
It might be useful to have a source-build expert audit our [dotnet-build](https://github.com/NuGet/NuGet.Client/tree/dev/eng/dotnet-build) folder.

Reply:
This shouldn't be passed on Windows.